### ISSUE TITLE
Can't compile master branch on Linux 

### ISSUE THREAD

**Issue by mbrlabs** [godotengine/godot#39901]
**Godot version:**
27378524f20234d6f0d4c262c514101df340c3b6

**OS/device including version:**
Manjaro Linux; gcc 10.1.0

**Issue description:**
Compilation aborts because of uninitialized variable in `drivers/vulkan/rendering_device_vulkan.cpp`
```
[ 44%] Compiling ==> editor/audio_stream_preview.cpp
[ 44%] Compiling ==> editor/code_editor.cpp
In file included from ./core/ustring.h:35,
                 from ./core/string_name.h:36,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/oa_hash_map.h:34,
                 from drivers/vulkan/rendering_device_vulkan.h:34,
                 from drivers/vulkan/rendering_device_vulkan.cpp:31:
./core/cowdata.h: In member function 'void RenderingDeviceVulkan::_draw_list_insert_clear_region(RenderingDeviceVulkan::DrawList*, RenderingDeviceVulkan::Framebuffer*, Point2i, Point2i, bool, const Vector<Color>&, bool, float, uint32_t)':
./core/cowdata.h:141:3: error: 'clear_at.VkClearAttachment::clearValue.VkClearValue::color.VkClearColorValue::float32[3]' may be used uninitialized in this function [-Werror=maybe-uninitialized]
  141 |   _get_data()[p_index] = p_elem;
      |   ^~~~~~~~~
drivers/vulkan/rendering_device_vulkan.cpp:5647:21: note: 'clear_at.VkClearAttachment::clearValue.VkClearValue::color.VkClearColorValue::float32[3]' was declared here
 5647 |   VkClearAttachment clear_at;
      |                     ^~~~~~~~
In file included from ./core/ustring.h:35,
                 from ./core/string_name.h:36,
                 from ./core/node_path.h:34,
                 from ./core/hashfuncs.h:36,
                 from ./core/oa_hash_map.h:34,
                 from drivers/vulkan/rendering_device_vulkan.h:34,
                 from drivers/vulkan/rendering_device_vulkan.cpp:31:
./core/cowdata.h:141:3: error: 'clear_at.VkClearAttachment::clearValue.VkClearValue::color.VkClearColorValue::float32[2]' may be used uninitialized in this function [-Werror=maybe-uninitialized]
  141 |   _get_data()[p_index] = p_elem;
      |   ^~~~~~~~~
drivers/vulkan/rendering_device_vulkan.cpp:5647:21: note: 'clear_at.VkClearAttachment::clearValue.VkClearValue::color.VkClearColorValue::float32[2]' was declared here
 5647 |   VkClearAttachment clear_at;
      |                     ^~~~~~~~
cc1plus: all warnings being treated as errors
scons: *** [drivers/vulkan/rendering_device_vulkan.linuxbsd.opt.tools.64.o] Error 1
scons: building terminated because of errors.
```

**Comment by Meriipu** [comment]
I have to add `werror=no` as a parameter to my scons command because of this

**Comment by mbrlabs** [comment]
Yeah, this is actually just a warning, but the build system treats warnings as errors by default.

**Comment by madmiraal** [comment]
Duplicate of #38829, which is fixed with #38835.
